backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5665090_sys_gh13959_Enable_Remote_Cache_For_GL_Journal.sql

```diff
@@ -0,0 +1,5 @@
+-- Table: GL_Journal
+-- 2022-11-17T16:43:57.810Z
+UPDATE AD_Table SET IsEnableRemoteCacheInvalidation='Y',Updated=TO_TIMESTAMP('2022-11-17 18:43:57','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Table_ID=224
```

Comment:
@metas-rc  FYI this is futile but leave it like that.

It's futile because de.metas.cache.model.WindowBasedCacheInvalidateRequestInitializer#setup it's automatically enabling remote cache invalidation for all tables which are main tab.